packages/gasket-plugin-webpack/lib/index.js

```diff
@@ -19,13 +19,15 @@ module.exports = {
           link: 'docs/webpack.md'
         }],
         lifecycles: [{
-          name: 'webpackChain (deprecated)',
+          name: 'webpackChain',
+          deprecated: true,
```

Comment:
I like this idea. It will affect what is displayed in the generated index table for the `gasket docs`, but that's something we could adjust for in the docs plugin.

Reply:
I'll make a follow up ticket for this ☝️